src/java.base/share/classes/java/nio/channels/DatagramChannel.java

```diff
@@ -655,7 +662,7 @@ public final long write(ByteBuffer[] srcs) throws IOException {
      * @return  The {@code SocketAddress} that the socket is bound to, or the
      *          {@code SocketAddress} representing the loopback address if
      *          denied by the security manager, or {@code null} if the
-     *          channel's socket is not bound
+     *          channel's socket is not bound.
```

Comment:
The period at the end of the return description isn't needed, the rest looks good.